app/newsheadlines.py

```diff
@@ -0,0 +1,51 @@
+import click
+import requests
+import os
+
+class NewsHeadlines:
+    def news_source_menu(self):
+        menu = ('Select your favorate source of News'+ '\n' + '1. FOX-News' + '\n' + '2. CNN' + '\n' + 
+            '3. Al- Jazeera - english'+ '\n' + '4. BBC-Sport'+ '\n' + '-' * 100)
+        return click.echo(click.style(menu, fg='blue'))
+        
+
+    def news_board(self):
+        board = ('\n'+ '*' * 100 + '*' + ' ' * 98 +' ' * 98 + '\n' +' ' * 30 + 
+                'News HeadLines ' + ' ' * 32 + '*' + '*' + '\n'+' ' * 12 + 'The latest and Live breaking news headlines ' 
+                + ' ' * 26 + '*' + '*' + ' ' * 98 + '*' + '*' + ' ' * 98 + '\n'+'*' + '*' * 100 + '\n')
+        return click.echo(click.style(board, fg='blue'))
+    
+    def get_key(self):
+        os.environ.get('API_KEY')
+        if 'API_KEY' not in os.environ:
+            raise ValueError("Your Key is invalid! Access Denied")
+        return os.environ.get('API_KEY')
+    
+    def cli(self, source_number):
```

Comment:
IMO the CLI functionality shouldn't be part of this class.
It should be maybe out as a separate function.

A better approach would be creating a method that receives parameters and makes a request to the API to this class and use that class  when implementing the CLI functionality

Reply:
This class can even be moved to a separate file for modularity

Reply:
when I separate the file, I get issues with import multiple scripts from the same package.
```
import os.path
import sys
sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```
When I use that, only one test file works and the other complains of the same issue. Now I have put the two classes into one file so that I can import that one script.